frontend/case-map.ts

```diff
@@ -42,54 +35,120 @@ function makeMap(regions_data) {
     return x.toString();
   }
 
-  function get_locations(dict) {
-    return Object.keys(dict).map(function(country) {
-      return dict[country]["iso_alpha_3"];
-    });
+  const iso_key = "iso_a3";
+
+  var countries_json = JSON.parse(
+    $.ajax({
+      url:
+        "https://storage.googleapis.com/static-covid/static/casemap-geo.json",
+      dataType: "json",
+      async: false
```

Comment:
Please don't use blocking requests

Reply:
Its better to add this to L210 and then pass it as an argumen to makeMap. There is something similar in `lines.ts`.

Reply:
Sorry, I spent a lot of time trying to load the json content to some variable, but I couldn't make it work without that `async: false` option. And I still don't know how to do that.